src/table/TableBodyWrapper.jsx

```diff
@@ -48,7 +59,8 @@ export default function TableBodyWrapper({ tableData, constraints, selectionsAPI
                 {value}
               </CellRenderer>
             ) : (
-              <TableCell key={column.id} align={column.align}>
+              // the new added column in the edit mode, both dimension and measure
+              <TableCell className={classes.body} key={column.id} align={column.align}>
```

Comment:
I think we should remove this. We always get a component from getCellRenderer, so this never runs